compiler/damlc/tests/daml-test-files/PreludeTest.daml

```diff
@@ -139,6 +139,7 @@ testFmap = scenario do
   [] === (T.length <$> [])
   Right 2 === fmap T.length (Right "10" : Either Text Text)
   Left "a" === fmap (+10) (Left "a")
+  (1, 3) === fmap T.length (1, "abc")
```

Comment:
CI is currently failing because line numbers have shifted. You need to update the expected diagnostics at the top of this file.

Reply:
thx - pretty annoying...